Tasks/AzureRmWebAppDeployment/Utility.ps1

```diff
@@ -119,7 +103,7 @@ function Get-MsDeployCmdArgs
     # msdeploy argument to exclude files in App_Data folder
     if($excludeFilesFromAppDataFlag -eq "true")
     {
-        $msDeployCmdArgs += [String]::Format(' -skip:objectname="dirPath",absolutepath="{0}\\App_Data$"', $webAppNameForMSDeployCmd)
+        $msDeployCmdArgs += [String]::Format(' -skip:objectname="dirPath",absolutepath="\\App_Data\\.*"')
```

Comment:
Did you try this in Azure website. Is this the right dirpath while deployed in kudu?